shell/browser/extensions/api/scripting/scripting_api.cc

```diff
@@ -498,7 +498,7 @@ ExtensionFunction::ResponseAction ScriptingExecuteScriptFunction::Run() {
   }
 
   std::string code_to_execute = absl::StrFormat(
-      "(%s)(%s)", injection_.func->c_str(), args_expression.c_str());
+      "(%s)(%s)", injection_.func.value_or(""), args_expression);
```

Comment:
Should this even execute when func is not provided ? Should we bail early with an error ?